_posts/jobs/2016-09-30-android.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Senior Android Engineer"
+description: We want you!
+modified:
+categories: jobs
+author: peter_krauss
+excerpt: We are looking for a Senior Android Engineer.
+tags: [Jobs, Android, Senior]
+date: 2016-09-30T10:39:55-04:00
+---
+## About the Role
+You are a Software Engineer by heart. You have clean code for breakfast sprinkled with MVVM or MVP, then you inject your dependencies while driving reactively to work. Then you integrate continuously and release it by lunchtime. In the afternoon you get up-to-date with the latest technologies and before leaving you improve the stress testing, add a espresso test or do whatever is necessary in order to have a monkey runner proof system you are proud of. You go to bed with the DoD under your pillow and dream of unit tests with a smile on your face.
+
+**You are:**
+
+* a person we could learn from.
+* interested in both, your quality of code and creating a great product
+* seeing problems yourself and take ownership solving them
+* striving to develop yourself and your team
+* cool to be around
+
+**Requirements (You have gathered professional experiences in):**
+
+* Android Core, RxJava and Dependency Injection
+* Gradle (Groovy)
+* Persistency Technologies (SQLite, Content Providers)
+* Test Driven Development (TDD - unit and UI)
+* Monitoring and maintaining the app (Fabric, Charles and Memory Profiling)
+* Continuous Integration/Continuous Delivery
+
+**Non functional Requirements**
+* BA/BS in Computer Science or equivalent practical experience.
+* You are a team player with experience working in an agile environment.
+* Willingness to develop your skills.
+* Understanding that professionalism is not the absence of fun and that absence of fun is unprofessional
+* An interesting hobby, like brewing beer or visiting Mars.
+
+We expect developing is not new to you. We will ask you to show your skills in these areas. We will also ask you to solve some problems for us to demonstrate your expertise.
+
+You may be young or you may be a veteran in software development. This is not the most important criteria but remember this is senior position, the bar is set high.
```

Comment:
Change `but remember this is senior position`, to `but remember, this is a senior position`